index.js

```diff
@@ -134,9 +134,29 @@ document.querySelectorAll('.btnModal').forEach((n) => n.addEventListener('click'
 
 // =============validtion form=============
 
-const email = document.getElementById('emailForm');
 const gontactForm = document.getElementById('contact-form');
 const messageform = document.querySelector('.messageform');
+const email = document.getElementById('emailForm');
+const name = document.getElementById('name');
+const msg = document.getElementById('msg');
+
+const dataForm = JSON.parse(localStorage.info) || [];
+
```

Comment:
- [ ] Great work attempting to make the changes suggested by the previous reviewer. However, the feature has not been implemented and still doesn't work. Remember that the requirements state that you should save all form data in an object not an array and save the object in the localStorage. Kindly implement this. 👍 

#### Hint: Delete this line of code